src/core/lib/attribute-manager.js

```diff
@@ -674,4 +688,15 @@ export default class AttributeManager {
       }
     }
   }
+
+  /**
+   * Update attribute transition to the current timestamp
+   * Returns updated attributes if any, otherwise `null`
+   */
+  updateTransition() {
+    const {attributeTransitionManger} = this;
+    const transitionUpdated = attributeTransitionManger.run();
```

Comment:
`run()`? Maybe not the best if this just updates to current time? (run feels like we are starting a loop) 

`setCurrentTime()`?